# file: C:\Users\atchi\AppData\Roaming\Python\Python311\site-packages\numpy\core\_exceptions.py
# hypothesis_version: 6.138.3

[1000, 1024, 'EiB', 'GiB', 'KiB', 'MiB', 'PiB', 'TiB', 'bytes', '{:#.0f} {}', '{:#.3g} {}', '{:.0f} {}', '{} ']